templates/challenge_forFirstConfig.php

```diff
@@ -0,0 +1,9 @@
+<img class="two-factor-icon" src="<?php print_unescaped(image_path('twofactor_email', 'email-code-light.svg')); ?>" alt="<?php p($l->t('Two-Factor Email app icon')); ?>">
+<p><?php p($l->t('A code has been sent to your email address.')) ?></p>
+<p><?php p($l->t('This verify your email for setup and enable this 2FA for your account')) ?></p>
```

Comment:
The english text should be improved. In my opinion, it should rather read "Verify your email address to enable Two-Factor Email (for your account)", and you may want to adapt the french translation accordingly.

But since the normal authentication dialog does not have such an additional explanation, I'd avoid it and not add an explanation at all.